CHANGELOG.rst

```diff
@@ -6,6 +6,156 @@
 Synapse Changelog
 *****************
 
+v2.145.0 - Unreleased
+=====================
+
+Automatic Migrations
+--------------------
+- Update indexing for light edges to index the N1 and N2 node identifiers
+  together.
+  (`#3302 <https://github.com/vertexproject/synapse/pull/3302>`_)
+- See :ref:`datamigration` for more information about automatic migrations.
+
+Model Changes
+-------------
+- Update to the ``inet``, ``it``, and ``meta`` models.
+  (`#3285 <https://github.com/vertexproject/synapse/pull/3285>`_)
+  (`#3298 <https://github.com/vertexproject/synapse/pull/3298>`_)
+  (`#3301 <https://github.com/vertexproject/synapse/pull/3301>`_)
+  (`#3310 <https://github.com/vertexproject/synapse/pull/3310>`_)
+
+  **New Types**
+
+  ``it:sec:tlp``
+    The US CISA Traffic-Light-Protocol used to designate information sharing
+    boundaries.
+
+  ``meta:priority``
+    A generic priority enumeration.
+
+  ``meta:severity``
+    A generic severity enumeration.
+
+
+  **New Forms**
+
+  ``it:sec:metrics``
+    A node used to track metrics of an organization's infosec program.
+
+  ``it:sec:vuln:scan``
+    An instance of running a vulnerability scan.
+
+  ``it:sec:vuln:scan:result``
+    A vulnerability scan result for an asset.``
+
+  **New Properties**
+
+  ``it:dev:repo:issue``
+    The form had the following properties updated on it:
```

Comment:
seems like all the entries in this section should be "added to it" rather than "updated on it"